src/backend/joanie/core/utils/payment_schedule.py

```diff
@@ -105,3 +110,30 @@ def generate(total, start_date, end_date):
     installments = _calculate_installments(total, due_dates, percentages)
 
     return installments
+
+
+@app.task
+def process_today_installment(order_id):
+    """
+    Process the payment schedule for the order.
+    """
+    Order = apps.get_model("core", "Order")  # pylint: disable=invalid-name
```

Comment:
I don't understand why you need to import the model this way ?

Reply:
It's for avoiding a cyclic import.

Reply:
So moving tasks to a dedicated module can solve the issue ? Something like this : 

```
core
  tasks
    __init__.py
    payment_schedule.py
```

Reply:
I put it directly in core/tasks.py

Reply:
Yes but you know we will have to split it later, so maybe we can start now ?